content/blog/2021-09-14-cats-and-dogs.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Switching the Dataset for Your Pre-trained Models
+date: 2021-09-14
+description: |
+  You can easily switch datasets for pre-trained models with DVC.
+descriptionLong: |
+  Sometimes your problem definition changes and you need to update your dataset.
```

Comment:
can we do a bit longer, informative descriptions? Ideally it would serve as an intro even or a summary of an intro.  We can apply styles here, emojis, etc.

Reply:
The descriptions are this short because usually we try to keep it > 80 characters. Do you want me to make the `descriptionLong` longer or just say something different and keep the same character count?

Reply:
< 80 usually means - wrap lines at 80 symbols. So it's per-line style convention, not for the `description` or `descriptionLong`. Sorry for the confusion here.

Reply:
Oh... Ok, that makes sense! No worries. Thanks for clearing that up.